backend/src/main/resources/application.yaml

```diff
@@ -78,6 +78,11 @@ management:
   endpoint.info.enabled: true
   endpoints.web.exposure.include: health, info
   endpoint.health.show-components: always
+  endpoint:
+    health:
+      status:
+        http-mapping:
+          okta_degraded: 204
```

Comment:
The other health status values [are automatically mapped to `200` and `503` respectively.](https://www.baeldung.com/spring-boot-health-indicators#5-health-status), so we just needed to add this one here because we're adding a custom one.

Reply:
if you'd want to add something to override the custom values lmk

Reply:
I was referring to the other config values in the lines above: `endpoint.health.probes.enabled: true` and `endpoint.health.show-components: always`

I'm assuming we can instead write them like this:
```suggestion
  endpoint:
    health:
      show-components: always
      status:
        http-mapping:
          okta_degraded: 204
      probes.enabled: true
```